kore/src/Kore/Repl/Data.hs

```diff
@@ -131,14 +136,21 @@ helpText =
     \exit                    exits the repl\
     \\n\
     \Available modifiers:\n\
-    \<command> > file        prints the output of 'command' to file\n"
+    \<command> > file        prints the output of 'command' to file\n\
+    \\n\
+    \(*) If an edge is labeled as Simpl/RD it means that\
+    \ either the next node was reached using the SMT\
+    \ solver or it was reached through the Remove Destination step."
 
 -- Type synonym for the actual type of the execution graph.
 type ExecutionGraph =
     Strategy.ExecutionGraph
         (CommonStrategyPattern Object)
         (RewriteRule Object Variable)
 
+type GraphType =
```

Comment:
I know I came up with this name initially, but how about `InnerGraph`? Not super useful for people reading for the first time, but at least it tells you something more.